mysite/todo/views.py

```diff
@@ -0,0 +1,5 @@
+from django.http import HttpResponse
+
```

Comment:
nitpick: put just one line between the imports and the first function definition

Reply:
Thank you